src/lib.rs

```diff
@@ -83,6 +89,127 @@ where
     }
 }
 
+// those functions are provided by the afl-llvm-rt static library
+extern "C" {
+    fn __afl_persistent_loop(counter: usize) -> isize;
+    fn __afl_manual_init();
+}
+
+/// Fuzz a closure by passing it a `&[u8]`
+///
+/// This slice contains a "random" quantity of "random" data.
+///
+/// ```rust,ignore
```

Comment:
Instead of `rust,ignore`, this can be `rust,no_run` to ensure it compiles, but won't attempt to run which is what we want

https://doc.rust-lang.org/book/first-edition/documentation.html#running-documentation-tests